docs/BR.md

```diff
@@ -1286,29 +1286,36 @@ No stipulation.
 
 ### 4.9.7 CRL issuance frequency
 
-CAs MUST generate and publish either:
-- a full and complete; OR
+CRLs must be available via a publicly-accessible HTTP URL (i.e., "published").
+
+Within 24 hours of issuing its first Certificate, CAs MUST generate and publish either:
+- a full and complete CRL; OR
 - partitioned (i.e., "sharded") CRLs, that when aggregated, represent the equivalent of the full and complete CRL.
 
-CRLs must be available via a publicly-accessible HTTP URL.
+CAs issuing Subscriber Certififcates MUST continue to issue CRLs in accordance with the table below:
+
+| __Conditions__ | __Minimum CRL Issuance Frequency__ | __Maximum nextUpdate Period__ | 
+| ----           | -                                  | -                             |
+| The CA has revoked at least one unexpired Certificate since the `thisUpdate` field of the last CRL. | A CRL MUST be generated and published within 24 hours of the `thisUpdate` field of the last CRL. <br><br> A CRL SHOULD be generated and published within 4 hours of the `thisUpdate` field of the current CRL. | The value of the `nextUpdate` field MUST NOT be more than 10 days beyond the value of the `thisUpdate` field. |
+| All other conditions. | A CRL MUST be generated and published within 7 days of the `thisUpdate` field of the last CRL. <br><br> A CRL SHOULD be generated and published within 4 hours of the `thisUpdate` field of the current CRL. | The value of the `nextUpdate` field MUST NOT be more than 10 days beyond the value of the `thisUpdate` field. |
```

Comment:
See above (abandoned this approach).

Justification for 4 hours was because I misunderstood discussion on thread. I thought people were interpreting Apple's policy ("_Under normal operating conditions, the CRL URLs provided by CAs in this section must be available such that Apple systems are able to successfully retrieve the current CRL every 4 hours._") to mean CRLs have to be updated every 4 hours (not the case).